File: src/main/java/com/gmail/borlandlp/minigamesdtools/gui/hotbar/HotbarCreatorHub.java
package com.gmail.borlandlp.minigamesdtools.gui.hotbar;

import com.gmail.borlandlp.minigamesdtools.creator.AbstractDataProvider;
import com.gmail.borlandlp.minigamesdtools.creator.CreatorHub;

public class HotbarCreatorHub extends CreatorHub {
    public Hotbar createHotbar(String ID, AbstractDataProvider dataProvider) throws Exception {
        return (Hotbar) this.create(ID, dataProvider);
    }
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/gui/hotbar/HotbarCreatorHub.kt
package com.gmail.borlandlp.minigamesdtools.gui.hotbar

import com.gmail.borlandlp.minigamesdtools.creator.AbstractDataProvider
import com.gmail.borlandlp.minigamesdtools.creator.CreatorHub

class HotbarCreatorHub : CreatorHub() {
    @Throws(Exception::class)
    override fun create(itemID: String, dataProvider: AbstractDataProvider): Hotbar {
        return super.create(itemID, dataProvider) as Hotbar
    }
}